flink-dist/pom.xml

```diff
@@ -113,8 +113,13 @@ under the License.
 			<artifactId>flink-metrics-jmx</artifactId>
 			<version>${project.version}</version>
 		</dependency>
+        
+		<dependency>
+			<groupId>org.apache.flink</groupId>
+			<artifactId>flink-mesos_2.10</artifactId>
+			<version>${project.version}</version>
+		</dependency>
```

Comment:
We might introduce an "include-mesos" profile where we add this dependency to flink-dist.